site/content/en/about/_index.html

```diff
@@ -0,0 +1,36 @@
+---
+title: About Kueue
+linkTitle: About
+menu:
+  main:
+    weight: 10
+
+---
+
+
+{{< blocks/cover title="Kueue" image_anchor="bottom" height="min" >}}
+
+{{< /blocks/cover >}}
+
+{{% blocks/lead %}}
+<h1 class="text-center">Kueue is a set of APIs and controller for job queueing.</h1>
+{{% /blocks/lead %}}
+
+
+{{< blocks/section >}}
+<div class="col-12">
+<h1 class="text-center">It is a job-level manager that decides when a job should be admitted to start (as in pods can be created) and when it should stop (as in active pods should be deleted).
+</h1>
+</div>
+
+{{< /blocks/section >}}
+
+
+
+{{< blocks/section >}}
+
+<div class="col-12">
+<h1 class="text-center">Kueue is a lean controller that you can install on top of a vanilla Kubernetes cluster. Kueue does not replace any existing Kubernetes components. </h1>
```

Comment:
I think it's better to copy everything from the main README

Reply:
Main readme has installation instruction that is not suited for an about page imo.